db/data_migration/20150317165803_republish_case_studies_to_render_on_government_frontend.rb

```diff
@@ -0,0 +1 @@
+DataHygiene::PublishingApiRepublisher.new(CaseStudy.where(:state => Edition::PUBLICLY_VISIBLE_STATES)).perform
```

Comment:
> @jamiecobbett: do we need to republish CaseStudy unpublishings within the data migration?

@vinayvinay I've made a stab at that below which seems to work, but I don't understand unpublishings that well. It found an additional 4 or 5 rows. Will this do the right thing?

```ruby
# Get published and unpublished case studies
case_studies = CaseStudy.where(
  "(state IN (?))
    OR
  ((state IN (?)) AND ((SELECT count(*) FROM unpublishings WHERE unpublishings.edition_id=editions.id) > 0))",
  Edition::PUBLICLY_VISIBLE_STATES,
  Edition::PRE_PUBLICATION_STATES
)
DataHygiene::PublishingApiRepublisher.new(case_studies).perform
```